install_dependencies.sh

```diff
@@ -22,7 +22,6 @@ FLAVOR=`grep '^ID=' /etc/os-release | awk -F= '{print $2}' | tr -d '"'`
 VERSION=`grep '^VERSION_ID=' /etc/os-release | awk -F= '{print $2}' | tr -d '"'`
 MAJOR=${VERSION%.*}
 ARCH=`uname -m`
-DEBIAN_FRONTEND="noninteractive"
```

Comment:
An alternative that should have the same impact is to `export` it here, rather than simply assigning.